tests/integration/test_line_minimization.py

```diff
@@ -26,5 +33,28 @@ def test_linemin(H2_ccecp_uhf):
     assert mfen > enfinal
 
 
+@pytest.mark.slow
+def test_linemin_three_body(H2_ccecp_uhf):
```

Comment:
Why is this testing linemin? Seems like this is actually testing the 3-body Jastrow and should be in a separate file.